src/current/v24.1/auto-tls.md

```diff
@@ -134,3 +134,19 @@ cockroach init --certs-dir=certs --host=localhost:26257
     ~~~
 
 1. Securely send each user the certificate and key that matches their username: `client.{username}.crt` and `client.{username}.key`.
+
+{{site.data.alerts.callout_info}}
+{% include {{page.version.version}}/misc/cert-auth-using-x509-subject.md %}
```

Comment:
Auto-tls is about using a cockroach-internal CA, so the x509 subject stuff isn't relevant to this file.

Reply:
Removed